packages/dynamite/dynamite/lib/src/helpers/dart_helpers.dart

```diff
@@ -17,7 +17,7 @@ String toDartName(
     }
   }
 
-  if (_reservedNames.contains(result) || RegExp(r'^[0-9]+$', multiLine: true).hasMatch(result)) {
+  if (_reservedNames.contains(result) || RegExp(r'^[0-9]+$', multiLine: true).hasMatch(result[0])) {
```

Comment:
Why this change?

Reply:
Not needed anymore as we now prefix all hashed names with a `$`.
I'll put it in a separate PR

Reply:
But then the expression will always be false because it never starts with a number?